de/optimize.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from scipy.optimize import least_squares
-from model import Model
-from importData import importRNAseqKO, formMatrix
+from .model import Model
+from .importData import importRNAseqKO, formMatrix
 
 def residual_fun(params, model, exp_data):
     model_data = model.sim(params)
```

Comment:
least_squares expects the difference between the model and data, not the sum of squares. You can use `X.flatten()` to turn it into a vector so it's the right shape.